src/Generator.jl

```diff
@@ -57,59 +78,63 @@ function gitignore()
     """
 end
 
-mkdocs_default(name, value, default) = value == nothing ? "#$name$default" : "$name$value"
-
 """
 $(SIGNATURES)
 
-Contents of the default `mkdocs.yml` file.
+Contents of the default `src/index.md` file.
 """
-function mkdocs(pkgname;
-        description = nothing,
-        author = nothing,
-        url = nothing
-    )
-    s = """
-    # See the mkdocs user guide for more information on these settings.
-    #   http://www.mkdocs.org/user-guide/configuration/
+function index(pkgname)
+    """
+    # $pkgname.jl
 
-    site_name:        $(pkgname).jl
-    $(mkdocs_default("repo_url:         ", url, "https://github.com/USER_NAME/PACKAGE_NAME.jl"))
-    $(mkdocs_default("site_description: ", description, "Description..."))
-    $(mkdocs_default("site_author:      ", author, "USER_NAME"))
+    Documentation for $pkgname.jl
 
-    theme: readthedocs
+    ```@index
+    ```
 
-    extra_css:
-      - assets/Documenter.css
+    ```@autodocs
+    Modules = [$pkgname]
+    ```
+    """
+end
 
-    extra_javascript:
-      - https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML
-      - assets/mathjaxhelper.js
+"""
+$(SIGNATURES)
 
-    markdown_extensions:
-      - extra
-      - tables
-      - fenced_code
-      - mdx_math
+Additions to `README.md`
+"""
+function readme(pkgname, user)
+    docs_stable_url = "https://$user.github.io/$pkgname.jl/stable"
+    docs_latest_url = "https://$user.github.io/$pkgname.jl/latest"
+    """
 
-    docs_dir: 'build'
+    ## Documentation
 
-    pages:
-      - Home: index.md
+    - [**STABLE**]($docs_stable_url) &mdash; **most recently tagged version of the documentation.**
+    - [**LATEST**]($docs_latest_url) &mdash; *in-development version of the documentation.*
     """
 end
 
 """
 $(SIGNATURES)
 
-Contents of the default `src/index.md` file.
+Additions to `travis.yml`
 """
-function index(pkgname)
+function travis(pkgname)
     """
-    # $(pkgname).jl
+      # build documentation
+      - julia -e 'ENV["DOCUMENTER_DEBUG"] = "true"; cd(Pkg.dir("$pkgname")); Pkg.add("Documenter"); include(joinpath("docs", "make.jl"))'
```

Comment:
If `DOCUMENTER_DEBUG` is going to be on by default, then probably best to just remove the option entirely in `deploydocs` and just always print the information.